pkg/mountmanager/fake-safe-mounter.go

```diff
@@ -39,8 +39,8 @@ func NewFakeNodeMounter() Mounter {
 // NewFakeSafeMounter ...
 func NewFakeSafeMounter() *mount.SafeFormatAndMount {
 	fakeMounter := &mount.FakeMounter{MountPoints: []mount.MountPoint{{
-		Device: "valid-devicePath",
-		Path:   "valid-vol-path",
+		Device: "some/target",
```

Comment:
Same Here

Reply:
This test was failing for Block too. Will raise another PR for sanity, which will work well for both file and block driver.